module/Entra/AdditionalFunctions/Get-EntraContext.ps1

```diff
@@ -1,18 +1,67 @@
 function Get-EntraContext{
```

Comment:
Add the MIT license. See this file
https://github.com/microsoftgraph/entra-powershell/blob/main/module/Entra/customizations/Add-EntraGroupOwner.ps1